src/logic/embedding.lean

```diff
@@ -232,6 +232,36 @@ end function
 
 namespace equiv
 
+/-- If `α₁ ≃ α₂` and `β₁ ≃ β₂`, then the type of embeddings `α₁ ↪ β₁`
+is equivalent to the type of enbeddings `α₂ ↪ β₂`. -/
+@[congr, simps apply] def embedding_congr {α β γ δ : Sort*}
+  (h : α ≃ β) (h' : γ ≃ δ) : (α ↪ γ) ≃ (β ↪ δ) :=
+{ to_fun := λ f,
+    ⟨h' ∘ f ∘ h.symm, (h'.comp_injective _).mpr $ (h.symm.injective_comp _).mpr f.injective⟩,
```

Comment:
Does something like this work?
```suggestion
    h.symm.to_embedding.trans (f.trans h'),
```

Reply:
yep, it works! also simplifies `embedding_congr_refl` to `ext, refl` :D